Entities/Items/Explosives/Explosion.as

```diff
@@ -69,10 +69,17 @@ void Explode(CBlob@ this, f32 radius, f32 damage)
 		{
 			//copy position, explode from centre of carrier
 			pos = doomed.getPosition();
-			//kill players if we're in their inventory (even water bombs for now)
+			//kill players if we're in their inventory
 			if ((doomed.hasTag("player") || doomed.getName() == "crate") && !doomed.hasTag("invincible"))
 			{
-				this.server_Hit(doomed, pos, Vec2f(), 100.0f, Hitters::explosion, true);
+				if (this.getName() == "bomb")
+				{
+					this.server_Hit(doomed, pos, Vec2f(), 100.0f, Hitters::explosion, true);
+				}
+				else if (this.getName() == "waterbomb")
+				{
+					this.server_Hit(doomed, pos, Vec2f(), 0.0f, Hitters::water_stun, true);
```

Comment:
Maybe `water_stun_force` instead? Not sure if that would make much of a difference.

Reply:
It doesn't seem to make any impact

Reply:
Even when shielding and with sponges in inventory?

Reply:
I didn’t think to test those